src/main/java/org/example/homework11/LetterCounter.java

```diff
@@ -0,0 +1,28 @@
+package org.example.homework11;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LetterCounter {
+    public static void main(String[] args) {
+        String inputString = "Homework 11 LetterCounter task";
+
+        countLetterOccurrences(inputString);
+    }
+
+    public static void countLetterOccurrences(String inputString) {
+        Map<Character, Integer> letterCountMap = new HashMap<>();
+
+        inputString = inputString.toLowerCase();
+
+        for (char c : inputString.toCharArray()) {
```

Comment:
```suggestion
        for (char c : inputString.toLowerCase().toCharArray()) {
```
minor: you can do `tolowercase()` right here